exercises/high-scores/.meta/HINTS.md

```diff
@@ -0,0 +1,5 @@
+In this exercise you're going to create a **class** and use lists to organize scores.  _Don't worry, it's not as complicated as you think it is!_ 
```

Comment:
```suggestion
In this exercise you're going to create a **class** and use lists to organize scores.  _Don't worry, it's not as complicated as you think!_ 
```

Don't need `it is` since phrase opens with `it's`.